ci/release_notes.md

```diff
@@ -1,10 +1,107 @@
+    before lockout.
+
+  - `uaa_lockout_failure_time_between_failures` - How much time
+    (in seconds) in which `uaa_lockout_failure_count` must occur in 
+    order for account to be locked. Defaults to `1200`.
+  
+  - `uaa_lockout_punishment_time` - How long (in seconds) the account
+    is locked out for violating `uaa_lock_failure_count` within 
+    `uaa_lockout_failure_time_between_failures`. Defaults to `300`.
+
+  - `uaa_refresh_token_validity` - How long (in seconds) a CF refresh
+    is valid for. Defaults to `2592000`.
+
+  - `cf_branding_product_logo` - A base64 encoded image to display on
+    the web UI login prompt. Defaults to `nil`.
+
+  - `cf_branding_square_logo` - A base64 encoded image to display
+    in areas where a smaller logo is necessary. Defaults to `nil`.
+
+  - `cf_footer_legal_text` - A string to display in the footer,
+    typically used for compliance text. Defaults to `nil`.
+
+  - `cf_footer_links` - A YAML list of links to enumerate in the footer
+    of the web UI. Defaults to `nil`
+
+# Upgrade Instructions
+
+From `1.0.0`, the following actions need to be made:
+
+## If CF was deployed with `local-db` or `local-ha-db` feature flags
+
+No changes need to be performed for an in-situ upgrade of 
+`cf-genesis-kit` 1.0.0 to 1.1
+
+## If CF was deployed with `mysql-db` or `postgres-db` feature flags
+
+1. Create 3 databases with the following names: `silkdb`, `locketdb`, and 
+  `policyserverdb`.
+
+
+
+# Core Components
+
+| Release | Version | Release Date |
+| --------- | ------- | ------------ |
+bpm (new) | [0.6.0](https://github.com/cloudfoundry-incubator/bpm-release/releases/tag/v0.6.0) | 2 May 2018
+capi | [1.60.0](https://github.com/cloudfoundry/capi-release/releases/tag/1.60.0) | 15 Jun 2018
+cf-smoke-tests | 40.0.5 | 17 May 2018
+cf-networking (new) | [1.9.0](https://github.com/cloudfoundry/cf-networking-release/releases/tag/v1.9.0) | 15 Dec 2017
+cflinuxfs2 | [1.212.0](https://github.com/cloudfoundry/cflinuxfs2-release/releases/tag/v1.212.0) | 4 Jun 2018
+cf-syslog-drain (new) | [6.5](https://github.com/cloudfoundry/cf-syslog-drain-release/releases/tag/v6.5) | 3 May 2018
+consul | [193](https://github.com/cloudfoundry-incubator/consul-release/releases/tag/v193) | 29 May 2018
+diego | [2.8.0](https://github.com/cloudfoundry/diego-release/releases/tag/v2.8.0) | 28 May 2018
+garden-runc | [1.14.0](https://github.com/cloudfoundry/garden-runc-release/releases/tag/v1.14.0) | 4 Jun 2018
+loggregator | [102.2](https://github.com/cloudfoundry/loggregator-release/releases/tag/v102.2) | 25 May 2018
+nats | [24](https://github.com/cloudfoundry/nats-release/releases/tag/v24) | 16 May 2018 
+cf-routing | [0.178.0](https://github.com/cloudfoundry/routing-release/releases/tag/0.178.0) | 17 May 2018
+statsd-injector | [1.3.0](https://github.com/cloudfoundry/statsd-injector-release/releases/tag/v1.3.0) | 23 Mar 2018
+uaa | [59](https://github.com/cloudfoundry/uaa-release/releases/tag/v59) | 22 May 2018
```

Comment:
Question (which may/should engender a new ticket):

Can/are we programmatically extracting this information from the kit yamls?

Reply:
Yes, please. That md list was the most tedious part of this entire PR. I can write a script that automatically generates something like this for future releases.